domains/emaeroles.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "emaeroles",
+        "email": "emanueleroles@gmail.com"
+    },
+    "record": {
+        "CNAME": "emaeroles.github.io"
```

Comment:
404 Site. Please send over a preview. Reopen this PR when you have this resolved.